easybuild/easyconfigs/c/ColabFold/ColabFold-1.5.5-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,187 @@
+    ('py3Dmol', '2.1.0'),
+
+
+    ('PyYAML', '6.0'),
+    ('TensorFlow', '2.13.0'),
+    ('Biopython', '1.83'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.4'),
+    ('Kalign', '3.4.0'),
+    ('jax', '0.4.25', versionsuffix),
+    ('UCX-CUDA', '1.14.1', versionsuffix),
+    ('cuDNN', '8.9.2.26', versionsuffix, SYSTEM),
+    ('NCCL', '2.18.3', versionsuffix),
+    ('OpenMM', '8.0.0', versionsuffix),
+    ('dm-haiku', '0.0.12', versionsuffix),
+    ('dm-tree', '0.1.8'),
+]
+
+local_scp_commit = '7102c6'
+
+components = [
+    ('stereo_chemical_props', '', {
+        'easyblock': 'Binary',
+        'source_urls': [
+            'https://git.scicore.unibas.ch/schwede/openstructure/-/raw/%s/modules/mol/alg/src/' % local_scp_commit,
+        ],
+        'sources': [{
+            'download_filename': 'stereo_chemical_props.txt',
+            'filename': 'stereo_chemical_props-%s.txt' % local_scp_commit,
+            'extract_cmd': "cp %s .",
+        }],
+        'checksums': ['24510899eeb49167cffedec8fa45363a4d08279c0c637a403b452f7d0ac09451'],
+    })
+]
+
+
+use_pip = True
+
+exts_list = [
+    ('PDBFixer', '1.9', {
+        'source_urls': ['https://github.com/openmm/pdbfixer/archive/refs/tags/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['88b9a77e50655f89d0eb2075093773e82c27a4cef842cb7d735c877b20cd39fb'],
+    }),
+    ('tabulate', '0.9.0', {
+        'checksums': ['0095b12bf5966de529c0feb1fa08671671b3368eec77d7ef7ab114be2c068b3c'],
+    }),
+    ('websocket-client', '1.5.1', {
+        'modulename': 'websocket',
+        'checksums': ['3f09e6d8230892547132177f575a4e3e73cfdf06526e20cc02aa1c3b47184d40'],
+    }),
+    ('docker', '7.0.0', {
+        'checksums': ['323736fb92cd9418fc5e7133bc953e11a9da04f4483f828b527db553f1e7e5a3'],
+    }),
+    ('immutabledict', '4.1.0', {
+        'checksums': ['93d100ccd2cd09a1fd3f136b9328c6e59529ba341de8bb499437f6819159fe8a'],
+    }),
+    ('contextlib2', '21.6.0', {
+        'checksums': ['ab1e2bfe1d01d968e1b7e8d9023bc51ef3509bba217bb730cee3827e1ee82869'],
+    }),
+    ('ml_collections', '0.1.1', {
+        'preinstallopts': "touch requirements.txt && touch requirements-test.txt && ",
+        'checksums': ['3fefcc72ec433aa1e5d32307a3e474bbb67f405be814ea52a2166bfc9dbe68cc'],
+    }),
+    ('AlphaFold', '2.3.6', {
+        'easyblock': 'PythonPackage',
+        'patches': [
+            'AlphaFold-2.3.2_data-dep-paths-shebang-UniRef30.patch',
+            'AlphaFold-2.0.0_n-cpu.patch',
+            'AlphaFold-2.0.1_setup_rm_tfcpu.patch',
+            'AlphaFold-2.3.2_BioPythonPDBData.patch',
+        ],
+        'preinstallopts': " cp %(builddir)s/stereo_chemical_props-7102c6.txt " +
+                          "alphafold/common/stereo_chemical_props.txt &&",
+        'runtest': "python ./run_alphafold_test.py",
+        'source_urls': [
+            'https://github.com/sokrypton/alphafold/archive/',
```

Comment:
@ThomasHoffmann77 I think it makes sense to add a comment here to explain why we're using a fork of `AlphaFold` here and bake it into the `ColabFold` installation, rather than just including `AlphaFold` as a dependency..